### ISSUE TITLE
Implement Pascal’s Triangle in CPP

### ISSUE THREAD

**Issue by swaroop-2021** [sobhanbera/algorithms#227]
#### This issue is about Pascal’s Triangle

-   [x] I searched or browsed the repo’s other issues to ensure this is not a duplicate

-   [x] Issue Title is in form Pascal’s Triangle in Cpp

-   [x] I will send my PR only after I'm assigned to this issue by any maintainer

-   [x] Assign this issue to me

-   [x] I ensure that I am not already assigned to 2 or more issues
  
Please Assign me this issue and add the hacktoberfest2023 tags, I am interested to Contribute : )

**Comment by bhaveshgarg2000** [comment]
Please Assign me this issue and add the hacktoberfest2023 tags, I am interested to Contribute : )

**Comment by shaileshthakur1** [comment]
Hey @swaroop-2021 , will you work on this issue or shall I assign this over to someone else?

**Comment by shaileshthakur1** [comment]
Hello @bhaveshgarg2000 
Issues are on FCFS basis. There are lot more algos to implement. You can open a new issue of your own as well and work on it.

**Comment by swaroop-2021** [comment]
Please Review my PR.
